cmd/mattermost/commands/command_test.go

```diff
@@ -169,3 +169,53 @@ func TestDeleteCommand(t *testing.T) {
 		assert.Error(t, th.RunCommand(t, "command", "delete", "invalid"))
 	})
 }
+
+func TestModifyCommand(t *testing.T)  {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+	team := th.BasicTeam
+	user := th.BasicUser
+	th.LinkUserToTeam(user, team)
+
+	originalCommand := &model.Command{
+		DisplayName: "dn_" + model.NewId(),
+		Method:      "G",
+		TeamId:      team.Id,
+		Username:    user.Username,
+		CreatorId:   user.Id,
+		Trigger:     "trigger_" + model.NewId(),
+		Description :"old description",
+		URL : "http://localhost:8000/modify-command",
+		IconURL : "original icon",
+		AutoComplete : false,
+	}
+	modifiedTitle := "modified title"
+	modifiedDescription := "modified description"
+	modifiedTrigger := "modified trigger"
+	modifiedURL := "http://localhost:8000/modified-command"
+	modifiedIcon := "modified icon"
+	modifiedUsername := "modified username"
+	modifiedCreator := "modified creator"
+
+	th.AddPermissionToRole(model.PERMISSION_MANAGE_SLASH_COMMANDS.Id, model.TEAM_USER_ROLE_ID)
+
```

Comment:
There is just one only test, and probably would be great to have a test table with more test, with things like, trigger with spaces or starting with `/`, invalid user, invalid command id, etcetera.

Reply:
I would love to have test to "not-passed" some parameters and modify only the parameters that were passed.

Reply:
I would add some more additional tests once my basic tests are passing. But looks like I am having issues even to pass this very basic test.

Reply:
@jespino Would you have any tips to offer for @akhilanandbv003 with unit tests?